readme/LinuxBuild.md

```diff
@@ -73,6 +73,10 @@ common available, therefore the Cairo PDF Export is without PDF Bookmarks.
 
 The binary executable will be placed in the `build/src/` subdirectory.
 
+To place the binary executable and support files in systemwide directories, execute `sudo make install`.
+
+After updating xournalpp, with perhaps `git pull`, execute `cmake ..`, `cmake --build .`, and optionally `sudo make install` again.
+
```

Comment:
These are steps that we explicitly discourage below.

Reply:
I did read that, but I don't understand why.  The procedure I mention works well for me.  Why shouldn't I do it?

Reply:
There is no clean way to uninstall after running `sudo make install`. If you upgrade or uninstall xournalpp, you risk leaving behind files that could cause problems down the road. Additionally, `sudo make install` is likely to be incompatible with your system's package manager.

There is a `rpm/fedora/xournalpp.spec` file if you would like to build an RPM package.

Reply:
OK, I made some changes.